bindings/rust/s2n-tls/src/testing/s2n_tls.rs

```diff
@@ -171,7 +171,7 @@ impl<'a, T: 'a + Context> Callback<'a, T> {
         let data = core::slice::from_raw_parts_mut(data, len as _);
         match context.on_read(data) {
             0 => {
-                // https://github.com/awslabs/s2n/blob/main/docs/USAGE-GUIDE.md#s2n_connection_set_send_cb
+                // https://aws.github.io/s2n-tls/doxygen/s2n_8h.html#a699fd9e05a8e8163811db6cab01af973
```

Comment:
This is the link to the correct section right now-- will it always be? Is this a predictable hash, or is it random / based on the order of methods in the file?

Reply:
Based on [this](https://github.com/doxygen/doxygen/issues/2567) the hash won't change unless we change the definition of the function. So this link won't break unless the API changes, which is unlikely.

Reply:
Perhaps this is overkill, but we could probably get the domain for something like `docs.s2n-tls.aws.dev` and then CNAME that to the doxygen site. This would also let get more human readable names for the things that host our CloudFront distributions with coverage reports, benchmark reports, etc.

Reply:
Instead of a hash is it possible to generate nicer human readable names for the url?